pkg/solidity-utils/test/foundry/StableMath.t.sol

```diff
@@ -68,6 +68,10 @@ contract StableMathTest is Test {
         amp = boundAmp(amp);
         uint256[] memory balances = boundBalances(rawBalances);
 
+        try stableMathMock.computeInvariant(amp, balances) returns (uint256) {} catch {
```

Comment:
Maybe a bit redundant, but perhaps we can check that the error was that the invariant didn't converge here and elsewhere.